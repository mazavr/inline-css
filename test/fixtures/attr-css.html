<html>
<head>
<style>
    p, td, img {
        -attr-width: 200;
    }
    p {
        height: 6px;
    }
</style>
</head>
<body>
    <p>none</p>
    <table>
        <tr>
            <td>
                wide
            </td>
        </tr>
    </table>
    <img src="" alt="wide">
</body>
</html>
