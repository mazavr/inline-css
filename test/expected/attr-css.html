<html>
<head>

</head>
<body>
    <p style="height: 6px;" width="200">none</p>
    <table>
        <tr>
            <td width="200">
                wide
            </td>
        </tr>
    </table>
    <img src="" alt="wide" width="200">
</body>
</html>
